workers/social/lib/social/models/session.coffee

```diff
@@ -39,7 +39,7 @@ module.exports = class JSession extends Model
       impersonating     : Boolean
       # data holds arbitary transitive session based info, not necessarily
       # should be used. There won't be any kind of index on this field.
-      data              : Object
+      sessionData       : Object
```

Comment:
hmm that's sound like it includes all the data related with the session but it only includes the custom/arbitrary data.